src/dbg/_exports.cpp

```diff
@@ -169,7 +169,7 @@ static bool getLabel(duint addr, char* label)
             duint start;
             if(FunctionGet(addr, &start, nullptr) && addr == start)
             {
-                sprintf_s(label, MAX_LABEL_SIZE, "sub_%" fext "X", start);

```

Comment:
Sorry I'm currently very ill but I will try to check your previous pull
request once I feel a bit better.
On Mon, 22 Aug 2016 at 11:07, Torusrxxx <notifications@github.com> wrote:

> In src/dbg/_exports.cpp
> <https://github.com/x64dbg/x64dbg/pull/998#discussion_r75641613>:
>
> > @@ -169,7 +169,7 @@ static bool getLabel(duint addr, char* label)
> >              duint start;
> >              if(FunctionGet(addr, &start, nullptr) && addr == start)
> >              {
> > -                sprintf_s(label, MAX_LABEL_SIZE, "sub_%" fext "X", start);
>
> Yes, it needs more fine tuning. Also the race condition is a major concern
> that could lead to crash. By the way, could you check the previous PR so we
> can get rid of those irrelavant diffs?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/x64dbg/x64dbg/pull/998/files/23241067481dd28ba1399c02a5cb5dca51a77d81#r75641613>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACWCmVfCwVkzbwAhSZ5y7shwexVF5fgFks5qiWa2gaJpZM4JpQ8m>
> .
>